src/Dotnet.Script.DependencyModel/Environment/RuntimeHelper.cs

```diff
@@ -11,22 +11,33 @@ public static class RuntimeHelper
         private static readonly Regex RuntimeMatcher =
             new Regex($"{GetPlatformIdentifier()}.*-{GetProcessArchitecture()}");
 
-        private static readonly Lazy<string> LazyTargetFramework = new Lazy<string>(GetNetCoreAppVersion); 
+        private static readonly Lazy<string> LazyTargetFramework = new Lazy<string>(GetNetCoreAppVersion);
 
-        public static string GetPlatformIdentifier()
+        private static readonly Lazy<string> LazyInstallLocation = new Lazy<string>(GetInstallLocation);
```

Comment:
can we package all that info into a type? I think it would make sense to have RuntimeEnvironment or something like, rather than many properties on the static helper. For now it could even be statically resolved from this helper, but would make more sense to me if things were organized that way

I can envision that in the future we will move away from the static altogether and pass around that instance

Reply:
This information is sort of static, but we could with little effort rename the whole `RuntimeHelper` class into `RuntimeInfo` and make it a singleton. 

The usage would then be 

```
RuntimeInfo.Instance.TargetFramework;
```

> `RuntimeInfo` is probably better so that we don't introduce confusion with the existing `Microsoft.DotNet.PlatformAbstractions.RuntimeEnvironment`

Alternatively we can call it `ScriptEnvironment`